README.md

```diff
@@ -124,3 +128,71 @@ interpreting them with HIBA.
 libhiba provides convenience functions to parse and extract HIBA extensions
 directly from a certificate, but it does not verify the certificate. It assumes
 the certificate was validated by a prior stage.
+
+### Example CA
+
+Once you have completed the
+[compilation instructions](https://github.com/google/hiba/blob/main/INSTALL.md#compilation)
+you will have a `hiba-gen` and `hiba-ca.sh` in your hiba directory. You can now
+create CA keys, grants, identities, grant eligibility, and sign host and user
+certificates with these extensions. The following assumes you are in your hiba
+directory.
+
+1. Choose a directory for the CA structure.
+  * `$ export dest=/tmp/hiba-ca`
+1. Generate a CA keypair.
+  * ```
+$ ./hiba-ca.sh -c -d "$dest" -- -N ""
+== Generating CA keys in /tmp/hiba-ca/ca
+== Done ==
+```
+1. Generate a test user keypair (alternatively you can copy existing keys to
+`$dest/users/`).
+  * ```
+$ ./hiba-ca.sh -c -d "$dest" -u -I testuser -- -N ""
+== Generating user keys in /tmp/hiba-ca/users/testuser
+== Done ==
+Identity created: Private: /tmp/hiba-ca/users/testuser
+                  Public:  /tmp/hiba-ca/users/testuser.pub
+```
+1. Copy the host public key from the OpenSSH server to the `$dest/hosts`
```

Comment:
The keys you generated right above are user keys, not host keys. OpenSSH makes the difference between the two types.

If you meant to retrieve the key from the host, you should add a step, otherwise it might confuse people. Also, the import has to be done with higba-ca.sh -i -h -f path_to_pubkey -I hostname

Reply:
I changed to "Copy the OpenSSH server host public key from the OpenSSH server"...

My immediate hope is for this to be useful to vendors of networking devices, so how that is done differs based on the platform.

It all works fine for me simply copying files to the hosts structure with devicename.pub - I can sign those into certificates with the right identity information. That seems less steps than copying down the file to some path, keeping track of that, then using hiba-ca.sh to import. What am I missing?

Reply:
The eligibility requires a specific folder named after the principal, which is created when either creating keys or importing them using the hiba-ca.sh command.

I would also avoid external assumptions on hard-coded paths except for the HIBA policy itself.

Reply:
Interesting though on the folder name - no idea how I got it to work in my testing without the import.

Reply:
There is no "eligible"  for hosts, so it will most likely work, but not for users.